🌊 Stream overview page

flash1293 opened the pull request:
Stacked on https://github.com/elastic/kibana/pull/204004

<img width="1275" alt="Screenshot 2024-12-12 at 17 19 58" src="https://github.com/user-attachments/assets/2ad14305-15c0-4522-8e70-5691c50e381b" />

Adds some bits to the stream overview page:
* Number of docs for the current time range (let's stop here and don't build more of Kibana)
* List of child streams for wired streams
* Quick links tab (currently empty)


<!--ONMERGE {"backportTargets":["8.x"]} ONMERGE-->

<!--ONMERGE {"backportTargets":["8.x"]} ONMERGE-->

elasticmachine commented:
<!-- buildkite-pr-bot-trigger-comment -->:robot: Jobs for this PR can be triggered through checkboxes. :construction: 

:information_source: To trigger the CI, please tick the checkbox below :point_down: 

- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-pull-request --> Click to trigger **kibana-pull-request** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-project-from-pr --> Click to trigger **kibana-deploy-project-from-pr** for this PR!

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/264406)
* Commit: 3dbc12f41b6055adb09112a206f718d2009901ae

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/264406#01943c55-176d-4440-9366-ea5cb1937efd)

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/264406","number":264406,"commit":"3dbc12f41b6055adb09112a206f718d2009901ae"}],"number":264406}
buildkite-pr-comment-->

simianhacker reviewed (approved):
Looks good... I'd probably put "Tooltips" on the little icon controls but we can do that in a follow up.

flash1293 commented:
Thanks @simianhacker , added the tooltips. I also fixed a `managed: true` that snuck in in what looks like a merge conflict: https://github.com/elastic/kibana/pull/204079/files#diff-18005311ea05fa61fa5ec13ed7e658bbebc45afd7cdb39d68924379668a3c151L225

Should be good now.

flash1293 commented:
I'll merge on green UI

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/265183)
* Commit: 9926c3ca104396f4a6abad3d698f88f9441ee2e1

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/265183#01944653-84f8-43a7-bc6a-708dbe758588)

### History
* :broken_heart: [Build #264406](https://buildkite.com/elastic/kibana-pull-request/builds/264406) failed 3dbc12f41b6055adb09112a206f718d2009901ae

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/265183","number":265183,"commit":"9926c3ca104396f4a6abad3d698f88f9441ee2e1"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/264406","number":264406,"commit":"3dbc12f41b6055adb09112a206f718d2009901ae"}],"number":265183}
buildkite-pr-comment-->

kibanamachine commented:
Starting backport for target branches: 8.x

https://github.com/elastic/kibana/actions/runs/12675940351

kibanamachine commented:
Starting backport for target branches: 8.x

https://github.com/elastic/kibana/actions/runs/12713346506
